### ISSUE TITLE
XPPen XP-8060c: Pressure and Precision Issue

### ISSUE THREAD

**Issue by after-shower** [mypaint/mypaint#743]
#### Description of the problem

MyPaint couldn't emulate pressure and there's a large offset on writing over the canvas.
I am using XPPen XP-8060c.
![ezgif com-3cf1a91f52](https://cloud.githubusercontent.com/assets/13714661/20372961/dec27366-aca9-11e6-896e-f633654355f1.gif)

ScreenShots:
![2016-11-17 09_26_41-preferences](https://cloud.githubusercontent.com/assets/13714661/20372974/fc68df18-aca9-11e6-8a4c-c52e40141fc9.png)
![2016-11-17 09_27_25-pen tablet](https://cloud.githubusercontent.com/assets/13714661/20372975/fc6c12f0-aca9-11e6-9f88-7f77126ba1f7.png)
![2016-11-17 09_27_47-about mypaint w32](https://cloud.githubusercontent.com/assets/13714661/20372976/fc6d5020-aca9-11e6-9a5e-3f95e24a79e5.png)
![2016-11-17 09_32_23-mypaint](https://cloud.githubusercontent.com/assets/13714661/20372977/fc707af2-aca9-11e6-93a7-f108fa49c51f.png)

Using uneven dual screen:
![2016-11-17 09_42_22-screen resolution](https://cloud.githubusercontent.com/assets/13714661/20373003/3f6fa5ee-acaa-11e6-881e-a8f532851516.png)

#### Basic system details

MyPaint version: 1.2.1-beta.1+gitexport.3198e3f
Operating system: Windows 7 x64
Tablet: XPPen XP-8060c

#### Steps to reproduce

1. Start MyPaint
2.  Just write on the canvas

**Comment by achadwick** [comment]
Couple of quick questions:

* Does the offset issue go away when you close MyPaint, remove your second monitor, and restart MyPaint
* Can you follow the instructions at https://github.com/mypaint/mypaint/wiki/Debugging-tablet-issues-on-Windows and paste in the debug output? Thanks.

**Comment by achadwick** [comment]
@after-shower 
Please respond.

[MyPaint v1.2.1](https://github.com/mypaint/mypaint/releases/tag/v1.2.1) was released recently. Please test it out, and let me know whether this problem is still happening.

**Comment by after-shower** [comment]
- The problem is fixed whenever i disconnect my second monitor, then restart MyPaint.
- The newer versions of MyPaint doesn't even detect the pressure from my tablet. (MyPaint 1.0.0 detects pressure)

- (I am still reading and working on the new Self-Diagnostics. seems to be pretty hard to follow but i've been reading it up and will update soon.)

**Comment by achadwick** [comment]
@after-shower
Sorry, that's a little unclear. Are you describing what happens with the current build of 1.2.1? MyPaint version 1.0.0 uses a much earlier version of the GTK libraries, and support has dried up for that. We do want the current series to work properly though, so I'll do what I can to make a good upstream report.

:link: https://github.com/mypaint/mypaint/wiki/Debugging-tablet-issues-on-Windows

I'm also keeping an eye on <https://git.gnome.org/browse/gtk+/log/?h=gtk-3-22> to see when a new version of GTK is released. I want to do a new MyPaint build with that because it contains a number of bugfixes for tablet issues.

**Comment by AesaraB** [comment]
Issue closed as stale. Possible related issues are #763 and #1150
